esmf_regrid/experimental/io.py

```diff
@@ -0,0 +1,150 @@
+"""Provides load/save functions for regridders."""
+
+import iris
+from iris.coords import AuxCoord
+from iris.cube import Cube, CubeList
+from iris.experimental.ugrid import PARSE_UGRID_ON_LOAD
+import numpy as np
+import scipy.sparse
+
+from esmf_regrid.experimental.unstructured_scheme import (
+    GridToMeshESMFRegridder,
+    MeshToGridESMFRegridder,
+)
+
+
+def save_regridder(rg, filename):
+    """
+    Save a regridder scheme instance.
+
+    Saves either a `GridToMeshESMFRegridder` or a `MeshToGridESMFRegridder`.
+
+    Parameters
+    ----------
+    rg : GridToMeshESMFRegridder, MeshToGridESMFRegridder
+        The regridder instance to save.
+    filename : str
+        The file name to save to.
+    """
+    src_name = "regridder source field"
+    tgt_name = "regridder target field"
+    if isinstance(rg, GridToMeshESMFRegridder):
+        regridder_type = "grid to mesh"
+        src_grid = (rg.grid_y, rg.grid_x)
+        src_shape = [len(coord.points) for coord in src_grid]
```

Comment:
Shame you can't do `src_grid.shape`... or can you?

Might be a handy property to have, as it'll avoid such boiler-plate code as above.

(not necessary to fix for this PR btw - just an observation 👍)

Reply:
Well at this point I think `src_grid` is just a tuple of coords so I don't believe it would have a `.shape`.